src/azure-cli/azure/cli/command_modules/network/operations/watcher.py

```diff
@@ -378,7 +366,89 @@ def _build_arguments_schema(cls, *args, **kwargs):
         return args_schema
 
     def pre_operations(self):
-        update_network_watcher_from_location(self.ctx,
-                                             self.cli_ctx,
-                                             watcher_name='network_watcher_name',
-                                             rg_name='resource_group_name')
+        get_network_watcher_from_location(self,
+                                          watcher_name='network_watcher_name',
+                                          rg_name='resource_group_name'
+                                          )
+
+
+def process_nw_cm_v2_output_namespace(cmd):
+    args = cmd.ctx.args
+    v2_output_optional_parameter_set = ['workspace_id']
+    if hasattr(args, 'out_type') and args.out_type.to_serialized_data() is not None:
```

Comment:
Please use has_value(args.out_type) if use not define args.out_type. The value of arg can be `AAZUndefined` if the user does not define this argument or `None` of user want to clear this property. The `has_value` functino is used to detect `AAZUndefined`.